lang/de/availability_shibboleth2fa.php

```diff
@@ -15,34 +15,36 @@
 // along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
 
 /**
+ * German language strings for the plugin.
+ *
  * @package    availability_shibboleth2fa
  * @copyright  2021 Lars Bonczek, innoCampus, TU Berlin
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
 defined('MOODLE_INTERNAL') || die();
 
-$string['title'] = '2FA';
+$string['description'] = 'Erfordere eine Zwei-Faktor-Authentifizierung.';
+$string['eventuser2faloggedin'] = 'Nutzer/in hat sich mit einem zweiten Faktor authentifiziert';
 $string['fulltitle'] = 'Zwei-Faktor-Authentifizierung';
+$string['login_failed'] = 'Der Authentifizierungs-Vorgang ist fehlgeschlagen. Bitte kontaktieren Sie Ihren Server-Administrator oder versuchen Sie es später erneut.';
```

Comment:
Would rather write "Authentifizierungsvorgang" and "Administrator" or "Serveradministrator"

Reply:
Right, I just sorted those lines without reading any of the strings. :laughing: But I agree and will change this.

Reply:
Fixed with 0953ee0.